client/sites-dashboard/components/sites-rows-icon.tsx

```diff
@@ -0,0 +1,22 @@
+export const SitesRowsIcon = ( { className }: { className?: string } ) => {
```

Comment:
maybe ListIcon would be better name here? plays better with the List view and the Tile view.